nextcord/types/checks.py

```diff
@@ -34,7 +34,9 @@
 Coro = Coroutine[Any, Any, T]
 MaybeCoro = Union[T, Coro[T]]
 CoroFunc = Callable[..., Coro[Any]]
-ApplicationCheck = Callable[[Interaction], MaybeCoro[bool]]
+ApplicationCheck = Union[
+    Callable[[ClientCog, Interaction], MaybeCoro[bool]], Callable[[Interaction], MaybeCoro[bool]]
```

Comment:
Do these nested brackets not require a Union?

Reply:
They represent the arguments to the function

Reply:
In that case, shouldnt it be `List[..]`?